pkg/server/server_resolver.go

```diff
@@ -33,5 +37,24 @@ func (*Server) reflectEnabled(_ string) (slog.LogLevel, bool) {
 
 // ServeDNS handles dns requests based on the IP address of the client
 func (srv *Server) ServeDNS(rw dns.ResponseWriter, req *dns.Msg) {
+	var chaos bool
```

Comment:
It should be possible to explicitly disable the `CHAOS` protocol as some users might not want to reveal the server type

Reply:
What should we return if CHAOS is disabled? Not implemented or an empty Success?